dataprofiler/profilers/profile_builder.py

```diff
@@ -23,6 +23,7 @@
 from .helpers.report_helpers import calculate_quantiles, _prepare_report
```

Comment:
Can we see a picture of how it looks now?
Also, does this cause any slowdown?

Reply:
In terms of slowdown,
with progress bar on guns.csv file: `20.00s user 2.37s system 107% cpu 20.799 total`
without progress bar on guns.csv file: `19.84s user 2.37s system 107% cpu 20.603 total`

Reply:
<img width="1792" alt="Screen Shot 2021-03-22 at 12 38 02 PM" src="https://user-images.githubusercontent.com/24511654/112029781-fa987d80-8b0f-11eb-95ee-5cf948f801d8.png">

Reply:
ah I see, this was my fear. Looks kind of janky. But still seems to be an improvement. We'll have to look into that tensorflow warning.

Reply:
Yeah, for sure; there does seem to be an issue open for it though - https://github.com/capitalone/DataProfiler/issues/63